templates/vue/src/components/Lightbox/media/MultiContentMedia/PageMenu/index.vue

```diff
@@ -35,30 +35,27 @@
           },
         ]"
       >
-        <ul class="page-menu-item fa-ul">
-          <page-menu-item
-            v-for="row in rows"
-            :key="row.node.id"
-            :node="row.node"
-            :lockRows="lockRows"
-            :disabled="disabledRow(row.node)"
-            @scroll-to="scrollToRef"
-          />
-        </ul>
+        <menu-groups

```

Comment:
Most of our components are single items, so I think it would be better if this is a representation of a single menu group rather than all menu groups. That way, you can put the v-for on the single menu-group component and have the component only responsible for what's inside of that one menu group. This is generally the preferred pattern in Vue.